### ISSUE TITLE
Allow browsing by tag

### ISSUE THREAD

**Issue by fasterthanlime** [itchio/itch#130]
I'm partial to a chrome-like omnibar? Search for a tag, game, user, whatever, click, it opens a 'tab' with all the stuff

**Comment by joonjoonjoon** [comment]
yeah absolutely. this would possibly be one of my main uses: post-game-jam-game-browsing.
